time-clock/app/javascript/packs/timer-script.js

```diff
@@ -0,0 +1,34 @@
+window.onload = function () {
```

Comment:
It'd be nice if we can refactor this functionality to use Stimulus JS to follow the new Rails conventions